slsDetectorSoftware/CMakeLists.txt

```diff
@@ -13,6 +14,7 @@ add_library(slsDetectorObject OBJECT
 
 target_include_directories(slsDetectorObject PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../slsSupportLib/opensslMd5>"
```

Comment:
Having already built the md5 as a part of the support lib we could remove the include here.